### PR TITLE
feat(hook): add new hook for 163jiasu CDN host,aka xxxx.163jiasu.com

### PR DESCRIPTION
**Pull request by LittleRey**
add hook host:
apm3.music.163.com.163jiasu.com
apm.music.163.com.163jiasu.com
interface3.music.163.com.163jiasu.com
interface.music.163.com.163jiasu.com
music.163.com.163jiasu.com

with updata readme.md

### CODE REVIEW

**Review by pan93412 — APPROVED** [review]

**Comment by 1715173329** [comment]
I wonder why... These domains don't even have a resolve record.

EDIT: looks like only interface*.music.163.com.163jiasu.com is valid

**Comment by LittleRey** [comment]
> I wonder why... These domains don't even have a resolve record.
> 
> EDIT: looks like only interface*.music.163.com.163jiasu.com is valid

in some devices ,"interface.music.163.com.163jiasu.com" it's works

**Comment by 1715173329** [comment]
there's no `music.163.com.163jiasu.com` and `apm*.music.163.com.163jiasu.com`, please drop them

**Comment by LittleRey** [comment]
> there's no `music.163.com.163jiasu.com` and `apm*.music.163.com.163jiasu.com`, please drop them

ok, it's changed

File: README.md [hunk 1]
```diff
@@ -234,11 +234,15 @@ checknetisolation loopbackexempt -a -n="1F8B0F94.122165AE053F_j2p0p5q0044a6"
...
 ### 方法 1. 修改 hosts
 
-向 hosts 文件添加两条规则
+向 hosts 文件添加几条规则
 
 ```hosts
 <Server IP> music.163.com
 <Server IP> interface.music.163.com
+<Server IP> interface3.music.163.com
+<Server IP> music.163.com.163jiasu.com
```
Comment (1715173329) [inline README.md]:
```suggestion
```

File: src/hook.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ hook.target.host = new Set([
 	'interface3.music.163.com',
 	'apm.music.163.com',
 	'apm3.music.163.com',
+
+	// 'music.163.com.163jiasu.com',
```
Comment (1715173329) [inline src/hook.js]:
```suggestion
```

File: src/hook.js [hunk 3]
```diff
@@ -47,6 +47,12 @@ hook.target.host = new Set([
...
+	'interface.music.163.com.163jiasu.com',
+	'interface3.music.163.com.163jiasu.com',
+	// 'apm.music.163.com.163jiasu.com',
+	// 'apm3.music.163.com.163jiasu.com',
```
Comment (1715173329) [inline src/hook.js]:
```suggestion
```

**Review by 1715173329 — APPROVED** [review]
otherwise LGTM

**Pull request merged**